src/Sarif/VersionOne/Core/FileDataVersionOne.cs

```diff
@@ -49,7 +49,7 @@ public partial class FileDataVersionOne : ISarifNodeVersionOne
                 string filePath = uri.LocalPath;
                 bool encodeAsUtf8 = (fileData.MimeType != SarifWriters.MimeType.Binary);
 
-                if (loggingOptions.Includes(Writers.LoggingOptions.PersistFileContents))
+                if (loggingOptions.Includes(Writers.LoggingOptions.PersistTextFileContents))
```

Comment:
>PersistTextFileContents [](start = 67, length = 23)

we should zap these version one helpers.

Reply:
Yes. You'll recall I argued against including them at all, on the principle that the only valid use of the v1 OM was to convert it to v2 -- and so we shouldn't offer people v1 programming aids by way of these helpers.